src/Personnummer.php

```diff
@@ -123,6 +123,41 @@ public static function getAge($ssn, $includeCoordinationNumber = true)
         return $d1->diff($d2)->y;
     }
 
+    /**
+     * Check if a Swedish social security number is for a male.
+     *
+     * @param string|int $ssn
+     * @param bool       $includeCoordinationNumber
+     *
+     * @return bool
+     * @throws PersonnummerException
+     */
+    public static function isMale($ssn, $includeCoordinationNumber = true)
```

Comment:
Is the coordination number useful here? I think we could skip it as it won't make any difference to the last 4 digits either way :)